### PR TITLE
Consul secure on kubernetes

### PR DESCRIPTION
**Pull request by DerekStrickland**

### CODE REVIEW

File: consul-secure-on-kubernetes/provision.courseData.sh [hunk 1]
```diff
@@ -0,0 +1,70 @@
+cat << 'EOFSRSLY' > /tmp/provision.sh
```
Comment (topfunky) [inline consul-secure-on-kubernetes/provision.courseData.sh:1]:
```suggestion
cat << 'EOF' > /tmp/provision.sh
```

I like humor as much as the next person, but for public code let's stick to non-silly variable names like `EOF`

File: consul-secure-on-kubernetes/provision.courseData.sh [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+## ================================
+log "Cleaning temporary files"
+
+rm -rf /tmp/provision
+
+
+finish
+
+EOFSRSLY
```
Comment (topfunky) [inline consul-secure-on-kubernetes/provision.courseData.sh:68]:
```suggestion
EOF
```

File: consul-secure-on-kubernetes/step1.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
+<style>
```
Comment (topfunky) [inline consul-secure-on-kubernetes/step1.md:1]: It worries me to include inline, hard-coded CSS that is going to be difficult to maintain.

Is this important enough that we really need it, or can we use the built-in Katacoda styles?

Comment (DerekStrickland) [reply consul-secure-on-kubernetes/step1.md:1]: I did try to steal some of their css classes to varying degrees of success. I'll look for documentation on how to style the katacoda way.

Comment (topfunky) [reply consul-secure-on-kubernetes/step1.md:1]: I guess I meant that we should do a regular code block and not try to style it at all (but you may be trying to achieve something different).

Comment (DerekStrickland) [reply consul-secure-on-kubernetes/step1.md]: Reverted to katacoda plaintext.

**Pull request merged**